src/view/renderer.js

```diff
@@ -177,6 +185,10 @@ export default class Renderer {
 	 * removed as long selection is in the text node which needed it at first.
 	 */
 	render() {
+		if ( this.isComposing ) {
+			return;
```

Comment:
Looks like asking for serious troubles with collaboration. If you allow changing the view, but prevent changes in DOM, you may have a very different state of view and DOM if another user changed the same part of the document. Mutations calculates a diff between DOM and view and if there is something for the OT in the view it will be marked to be removed since it does not appear in the DOM.